File: machi_koro/cards/restaurants.py
import machi_koro.cards.establishment_base as establishment_base
import machi_koro.utils as utils
import machi_koro.card_effects.establishments.restaurants as card_effect_restaurants

from dataclasses import dataclass


@dataclass
class RestaurantsCard(establishment_base.EstablishmentBase):
    card_type: utils.CardType = utils.CardType.Restaurants
    card_color: utils.CardColor = utils.CardColor.RED
    card_icon: utils.CardIcon = utils.CardIcon.CUP
    card_effect: card_effect_restaurants = card_effect_restaurants.CardEffectRestaurants

File: MachiKoro.Core/Models/Cards/Establishments/Basic/Restaurants.cs
using MachiKoro.Core.CardEffects.Establishments.Basic;
using System.Collections.Generic;

namespace MachiKoro.Core.Cards.Establishments.Basic
{
    public class Restaurants : EstablishmentBase
    {
        private readonly RestaurantsCardEffect _restaurantsCardEffect;

        public Restaurants(string name, CardCategory cardIcon, List<int> activationNumbers, int constructionCost, RestaurantsCardEffect restaurantsCardEffect)
            : base(name, cardIcon, activationNumbers, constructionCost, restaurantsCardEffect)
        {
            _restaurantsCardEffect = restaurantsCardEffect;
        }
    }
}
